dev-lua/lua-resty-lrucache/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
We can't add new m-n packages: https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0601. Arguably, at least some of them could come under the "split out" exception, but ideally please adopt them for now at least until things are settled.

Reply:
(Answered on IRC)